app/src/main/java/org/mozilla/focus/theme/ThemeManager.kt

```diff
@@ -0,0 +1,14 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
```

Comment:
nit: Sorry, I forgot to call out that the class name should also be the class that we're adding extensions to. In this case `Theme.kt`.